includes/sanitizers/class-amp-iframe-sanitizer.php

```diff
@@ -135,8 +135,20 @@ public function sanitize() {
 				if ( $node->parentNode->parentNode instanceof DOMElement && 'figure' === $node->parentNode->parentNode->tagName ) {
 					$figure_node = $node->parentNode->parentNode;
 				}
-				if ( $figure_node && $figure_node->hasAttribute( 'class' ) && in_array( 'alignfull', explode( ' ', $figure_node->getAttribute( 'class' ) ), true ) ) {
-					$normalized_attributes['layout'] = 'responsive';
+
+				if ( $figure_node && $figure_node->hasAttribute( 'class' ) ) {
+
+					$figure_node_classes = explode( ' ', $figure_node->getAttribute( 'class' ) );
```

Comment:
It will be more robust to split by any whitespace because there could be newlines or other characters than spaces.

```suggestion
					$figure_node_classes = preg_split( '/\s+/', trim( $figure_node->getAttribute( 'class' ) ) );
```

Reply:
Thanks for the feedback. I have applied the suggested changes.